x-pack/plugin/sql/src/test/java/org/elasticsearch/xpack/sql/analysis/analyzer/VerifierErrorMessagesTests.java

```diff
@@ -97,7 +102,28 @@ public void testMisspelledColumnWithWildcard() {
     public void testColumnWithNoSubFields() {
         assertEquals("1:8: Cannot determine columns for [text.*]", error("SELECT text.* FROM test"));
     }
-    
+
+    public void testFieldAliasTypeWithoutHierarchy() {
+        Map<String, EsField> mapping = new LinkedHashMap<>();
+
+        mapping.put("fld", new EsField("fld", DataType.KEYWORD, emptyMap(), true));
```

Comment:
Do you need that in the test?

Reply:
Not anymore, it was used in the previous commit when the alias field was hidden.